DirectProgramming/DPC++FPGA/ReferenceDesigns/db/src/query11/query11_kernel.cpp

```diff
@@ -4,12 +4,11 @@
 
 #include "query11_kernel.hpp"
 #include "pipe_types.hpp"
-#include "../db_utils/Accumulator.hpp"
```

Comment:
You no longer use Accumulator.hpp in the Q11, but this PR includes changes to that file (as well as to other files in the db_utils directory).  Did you test all the db querries after your changes?

Reply:
Yes.